confluent updated dotnet SDK

alluri02 opened the pull request:
Swagger PR - Swagger PR is already merged - https://github.com/Azure/azure-rest-api-specs/pull/12362/

alluri02 committed:
added support for dotnet

alluri02 committed:
fix test case

alluri02 committed:
fix test cases

alluri02 committed:
fix test cases

alluri02 commented:
Hi, please review this PR. we are planning to have GA launch this week

allenjzhang commented:
Could you:

 - Add a link to the openapi spec review PR here

 - Update api tag in `AzSdk.RP.props`, release notes in `csproj` if any.

 - Update versions in `assemblyinfo.cs` 
 
Thanks

alluri02 committed:
code review comments

alluri02 commented:
> Could you:
> 
> * Add a link to the openapi spec review PR here
> * Update api tag in `AzSdk.RP.props`, release notes in `csproj` if any.
> * Update versions in `assemblyinfo.cs`
> 
> Thanks


Thanks Allen, updated the PR description with Swagger PR and also updated the versions
